src/combobox/combobox.component.ts

```diff
@@ -294,7 +299,7 @@ export class ComboBox implements OnChanges, OnInit, AfterViewInit, AfterContentI
 		} else if ((ev.key === "ArrowUp" || ev.key === "Up") // `"Up"` is IE specific value
 			&& this.dropdownMenu.nativeElement.contains(ev.target)
 			&& !this.view.hasPrevElement()) {
-			this.elementRef.nativeElement.querySelector(".bx--text-input").focus();
+			// this.elementRef.nativeElement.querySelector(".bx--text-input").focus();
```

Comment:
Do we want to completely remove this?

Reply:
yep, I think that was the intention ... oops 😬